src/graphql/resolvers/businessInfoResolver.ts

```diff
@@ -6,6 +6,51 @@ import {
 import { getInterpretationProvider, InterpretationConfig } from '../../providers/interpretation_providers';
 import { BusinessInferences, QueryGetBusinessInfoArgs } from '../generatedTypes';
 import { defaultConfig, DataSourceConfiguration } from '../../models/data_models/dataSourceConfiguration';
+import { BusinessInfoInput } from '../../models/data_models/types';
+import { GraphQLClient, gql } from 'graphql-request';
+
+async function fetchFromDB(businessInfo: BusinessInfoInput): Promise<BusinessInferences | undefined> {
+	const endpoint = 'http://host.docker.internal:3000/api/graphql';
```

Comment:
We can make this an environment variable

Reply:
Yes 🤡

Reply:
Yes